src/resources/css/parts-kit.css

```diff
@@ -65,7 +65,53 @@
 .parts-kit__sidebar .flex {
     display: flex;
 }
+/* Search */
+.parts-kit__search {
+    padding: 0 18px;
+    margin-bottom: 8px;
+    position: relative;
+}
+
+.parts-kit__search-icon {
+    fill: none;
+    border-color: var(--pk-text);
+    position: absolute;
+    top: -2px;
+    bottom: 0;
+    left: 18px;
+    margin: auto 0;
+    width: 16px;
+    height: 16px;
+    opacity: .5;
+}
+
+.parts-kit__search-input {
+    background: transparent;
```

Comment:
i kinda also wanna add `appearance: none` just bc i really dislike the blue-gradient "x" for clearing

Reply:
Yeah, I think you need a `border: none` as well

![image](https://user-images.githubusercontent.com/199985/106669320-7c8d1100-6579-11eb-868a-09f3a7b5d0bd.png)